script/client

```diff
@@ -244,6 +244,7 @@ GetOptions(
     'asset-size-limit:i', 'with-thumbnails', 'accept=s',    'yaml-output',
 ) or usage(1);
 
+usage(0) if $options{help};
```

Comment:
I think there is a self-test that checks for the exit code.

Reply:
What do you mean?

Reply:
Ok, I checked and I confused *client* with *clone-job*. t/40-openqa-clone-job.t has the check. Which makes me wonder if we should apply the same change for `openqa-clone-job`

Reply:
Ok, I will add it there too.

Reply:
done